src/client/Microsoft.Identity.Client/Cache/CacheSessionManager.cs

```diff
@@ -45,7 +45,7 @@ public async Task<MsalAccessTokenCacheItem> FindAccessTokenAsync()
             return await TokenCacheInternal.FindAccessTokenAsync(_requestParams).ConfigureAwait(false);
         }
 
-        public Task<Tuple<MsalAccessTokenCacheItem, MsalIdTokenCacheItem, Account>> SaveTokenResponseAsync(MsalTokenResponse tokenResponse)
+        public Task<(MsalAccessTokenCacheItem accessCacheItem, MsalIdTokenCacheItem tokenCacheItem, Account account)> SaveTokenResponseAsync(MsalTokenResponse tokenResponse)
```

Comment:
naming: `AccessTokenCacheItem`, `IdTokenCacheItem`
(because access and token are too overloaded)